pkg/apiserver/rest/dualwriter.go

```diff
@@ -260,3 +295,46 @@ func getName(o runtime.Object) string {
 	}
 	return accessor.GetName()
 }
+
+const dataSyncerInterval = 60 * time.Minute
```

Comment:
What do you think the chances are of this creating big spikes in db access?

Since our grafana pods mostly restart all on the same cadence (due to rcc), we could have hundreds or thousands of pods all syncing within a window of a couple minutes.

Reply:
That's a very good point. Would a jitter be enough to scatter background jobs across a 30 minute window?

```
minutes := 30
timeWindow := 30 * 60 * 1000 // 30 minutes 
jitterMilliseconds := time.Duration(rand.Int63n(int64(timeWindow)))
time.Sleep(jitterMilliseconds)
```

What do you think?

Reply:
Ya a bit of randomness would probably help. Could do it in a future PR also - just wanted to bring it up so were aware.